src/satosa/backends/oauth.py

```diff
@@ -331,3 +333,22 @@ def get_metadata_desc_for_oauth_backend(entity_id, config):
 
     metadata_description.append(description)
     return metadata_description
+
+
+def _get_metadata_to_decorate(config):
+    metadata_dict = {}
+    if "entity_info" in config:
+        entity_info = config["entity_info"]
+        if "ui_info" in entity_info:
+            ui_info = entity_info["ui_info"]
+            for name in ui_info.get("display_name", []):
+                if name[1] == "en":
+                    metadata_dict["client_name"] = name[0]
+                metadata_dict["client_name#" + name[1]] = name[0]
```

Comment:
is this a convention that is defined somewhere?
(meaning the `<attr>#{language}` notation)